core/src/main/java/hudson/model/ViewGroup.java

```diff
@@ -58,6 +59,26 @@
     Collection<View> getViews();
 
     /**
+     * Gets all the views in this group including nested views.
+     *
+     * @return
+     *      can be empty but never null.
+     *
+     * @since 2.55
```

Comment:
Please do not explicitly specify versions next time. `@since TODO` is preferable